Added an important note. (#2966)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Skype/SfBServer2019/plan/system-requirements.md

<a name="Software"> </a>

There are some things you're going to need to install or configure for any server running Skype for Business Server 2019. These are listed below, followed by additional requirements for specific server roles.

> [!IMPORTANT]

> > .Net Framework 4.8 is not supported by Skype for Business Server 2019.

Reviewer: ```suggestion
> .NET Framework 4.8 is not supported by Skype for Business Server 2019.
```

> > .Net Framework 4.8 is still not supported with Skype For Business 2019.

Reviewer 2: ```suggestion
> .Net Framework 4.8 is not yet supported by Skype for Business Server 2019.
```
